friends/src/components/FriendForm.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import "./FriendForm.css";
+import PropTypes  from "prop-types";
+
+class FriendForm extends React.Component{
+      constructor(){
+          super();
+          this.state={
+               id:null,
+               inputName: "",
+               inputAge:null,
```

Comment:
This being null will throw errors in your console. Instead you could have it as a default 0